ddtrace/opentelemetry/span.go

```diff
@@ -35,13 +37,34 @@ func (s *span) TracerProvider() oteltrace.TracerProvider        { return s.otelt
 func (s *span) AddEvent(_ string, _ ...oteltrace.EventOption)   { /*	no-op */ }
 func (s *span) RecordError(_ error, _ ...oteltrace.EventOption) { /*	no-op */ }
 
-func (s *span) SetName(name string) { s.SetOperationName(name) }
+func (s *span) SetName(name string) {
+	s.attributes[ext.SpanName] = strings.ToLower(name)
```

Comment:
Why change this? Can't we just continue to call `s.SetOperationName(name)` here? Or `s.SetOperationName(strings.ToLower(name))` if you need to

Reply:
If we set it, we can't retrieve it. So, later when we want to create operation name, we don't know if it was set explicitly, if it was overriden with an empty string, etc